src/pages/SignIn/constants.ts

```diff
@@ -0,0 +1,6 @@
+import { Credentials } from './types';
+
+export const INITIAL_VALUES: Credentials = {
+  email: '',
```

Comment:
```suggestion
  [FormField.email]: '',
```
After implementing typings for fields